SysBatcher.py

```diff
@@ -20,7 +37,7 @@
 
 
 # Loop over channels and systematic variations
-for c in ['uujj','uvjj']:
+for c in ['uujj']:
```

Comment:
please just put back the original line commented out so we remember how to add it back later